types/repayments.ts

```diff
@@ -1,105 +1,36 @@
-import { BaseCreateRequestAttributes, Relationship, Tags } from "./common"
+import { BaseCreateAchRepaymentRequest, BaseCreateBookRepaymentRequest, BaseCreateCapitalPartnerAchRepaymentRequest, BaseCreateCapitalPartnerBookRepaymentRequest } from "./baseRepaymentTypes"
+import { Relationship, Tags } from "./common"
 
 type BaseCreateRepaymentRequest = {
-    type: string
-
     attributes: {
-        /**
-         * Payment description (maximum of 50 characters), also known as Company Entry Description, this will show up on statement of the counterparty.
-         */
-        description: string
-
-        /**
+         /**
          * The amount (Cents) to repay.
          */
-        amount: number 
-    } & BaseCreateRequestAttributes
-
-    relationships: {
-        /**
-         * The [Credit Account](https://www.unit.co/docs/api/resources/#credit-account) that the repayment is made against.
-         */
-        creditAccount: Relationship
+         amount: number 
     }
```

Comment:
I feel like there are too many "bases", especially when you go down everything you get to "BaseCreateRepaymentRequest" which has one attribute (amount), we're not sparing anything for ourselves besides making the code a little more convoluted.. The goal is not to find all common attributes and create a type for them, but to make life a little easier for us and in this case it's a little too much.

I would remove BaseCreateRepaymentRequest (just use "amount" on the things that use it)